src/SavedSearches/OwnershipSavedSearchRepository.php

```diff
@@ -66,9 +67,14 @@ private function getOwnershipQueryStrings(): array
 
         foreach ($ownershipItemCollection as $ownershipItem) {
             $organizerId = $ownershipItem->getItemId();
-            $organizerName = ExtractOfferName::extract(
-                $this->organizerDocumentRepository->fetch($organizerId)->getAssocBody()
-            );
+            $organizer = $this->organizerDocumentRepository->fetch($organizerId)->getAssocBody();
+
+            if ($organizer['workflowStatus'] === WorkflowStatus::DELETED()->toString() ||
```

Comment:
`src/Model/ValueObject/Moderation/WorkflowStatus.php` are the Workflowstatuses for `Offer`.
The ones for `Organizer` are in `src/Model/ValueObject/Moderation/Organizer/WorkflowStatus.php`
This will work because both have `DELETED`(but not `REJECTED`)but it's best to use the correct `WorkflowStatus`.